### PR TITLE
fix: Add server-side setup check note to upgrade_to_28

### PR DESCRIPTION
**Pull request by joshtrichards**
### ☑️ Resolves

In hindsight we probably should have added a note for this a long time ago. Oops.

Currently mobile. I'll try to remember to add a screenshot later.

### CODE REVIEW

File: admin_manual/release_notes/upgrade_to_28.rst [hunk 1]
```diff
@@ -18,6 +18,27 @@ Web server configuration
   * For Nginx, please refer to our recommended :ref:`Nginx configuration<nginx-config>`.
   * For other setups, make sure to add ``.mjs`` to the list of static file extensions in web server configs and in case define its MIME type in ``/etc/mime.types``.
 
+Setup Checks
+------------
+
+The setup checks (the ones visible under *Administration settings->Overview*) that previously ran from the web browser now run server-side rather than from the browser.
```
Comment (tflidd) [inline admin_manual/release_notes/upgrade_to_28.rst:24]: This is new in NC 28? Or was the change even older?

Comment (kesselb) [reply admin_manual/release_notes/upgrade_to_28.rst:24]: Changed for 28

**Review by kesselb — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by enoch85** [comment]
Sorry, wrong PR. :|

**Pull request merged**